docs/en/install-upgrade/air-gapped-install.asciidoc

```diff
@@ -120,7 +120,7 @@ Air-gapped setup of the APM server is possible in two ways:
 [[air-gapped-elastic-maps-service]]
 ==== 1.8. {ems}
 
-To learn about air-gapped setup of the {ems}, refer to {kibana-ref}/maps-connect-to-ems.html#elastic-maps-server[Host {ems} locally] in the {kib} documentation.
+Refer to {kibana-ref}/maps-connect-to-ems.html[Connect to {ems}] in the {kib} documentation to learn about how to configure your firewall to connect to {ems}, host it locally, or disable it completely.
```

Comment:
Looks great. Only recommendation is to remove the word "about" so "to learn how to configure" instead of "to learn about how to configure"

Reply:
Thanks Nathan! Fixed on b312698.